include/openssl/ssl.h

```diff
@@ -2039,6 +2039,10 @@ __owur const struct openssl_ssl_test_functions *SSL_test_functions(void);
 
 extern const char SSL_version_str[];
 
+
+void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timer));
+
+
```

Comment:
There's a bit too many empty lines added.  One after the declaration of `DTLS_set_timer_cb` is enough.